gensim/models/atmodel.py

```diff
@@ -0,0 +1,945 @@
+        if corpus is None and self.id2word is None:
+            raise ValueError('at least one of corpus/id2word must be specified, to establish input space dimensionality')
+
+        if self.id2word is None:
+            logger.warning("no word id mapping provided; initializing from corpus, assuming identity")
+            self.id2word = utils.dict_from_corpus(corpus)
+            self.num_terms = len(self.id2word)
+        elif len(self.id2word) > 0:
+            self.num_terms = 1 + max(self.id2word.keys())
+        else:
+            self.num_terms = 0
+
+        if self.num_terms == 0:
+            raise ValueError("cannot compute the author-topic model over an empty collection (no terms)")
+
+        logger.info('Vocabulary consists of %d words.', self.num_terms)
+
+        self.author2doc = {}
+        self.doc2author = {}
+
+        self.distributed = distributed
+        self.num_topics = num_topics
+        self.num_authors = 0
+        self.chunksize = chunksize
+        self.decay = decay
+        self.offset = offset
+        self.minimum_probability = minimum_probability
+        self.num_updates = 0
+        self.total_docs = 0
+
+        self.passes = passes
+        self.update_every = update_every
+        self.eval_every = eval_every
+
+        self.author2id = {}
+        self.id2author = {}
+
+        self.serialized = serialized
+        if serialized and not serialization_path:
+            raise ValueError("If serialized corpora are used, a the path to a folder where the corpus should be saved must be provided (serialized_path).")
+        if serialized and serialization_path:
+            assert not isfile(serialization_path), "A file already exists at the serialization_path path; choose a different serialization_path, or delete the file."
+        self.serialization_path = serialization_path
+
+        # Initialize an empty self.corpus.
+        self.init_empty_corpus()
+
+        self.alpha, self.optimize_alpha = self.init_dir_prior(alpha, 'alpha')
+
+        assert self.alpha.shape == (self.num_topics,), "Invalid alpha shape. Got shape %s, but expected (%d, )" % (str(self.alpha.shape), self.num_topics)
+
+        if isinstance(eta, six.string_types):
+            if eta == 'asymmetric':
+                raise ValueError("The 'asymmetric' option cannot be used for eta")
+
+        self.eta, self.optimize_eta = self.init_dir_prior(eta, 'eta')
+
+        self.random_state = utils.get_random_state(random_state)
+
+        assert (self.eta.shape == (self.num_terms,) or self.eta.shape == (self.num_topics, self.num_terms)), (
+                "Invalid eta shape. Got shape %s, but expected (%d, 1) or (%d, %d)" %
+                (str(self.eta.shape), self.num_terms, self.num_topics, self.num_terms))
+
+        if not distributed:
+            self.dispatcher = None
+            self.numworkers = 1
+        else:
+            # NOTE: distributed processing is not implemented for the author-topic model.
+            pass
```

Comment:
I don't know why I put that there, `distributed=False` is set explicitly (it is not an input). So now I just do the following:
```
distributed = False
self.dispatcher = None
self.numworkers = 1
```